doc/src/tutorials/physics/quantum_mechanics/creation_annihilation_tutorial.rst

```diff
@@ -0,0 +1,39 @@
+.. _creation_annihilation_tutorial:
+
+======================================
+Creation and Annihilation Operators
+======================================
+
+This tutorial demonstrates the algebra of creation and annihilation operators using
+the :mod:`sympy.physics.quantum` tools.
+
+.. code-block:: python
+
+   from sympy.physics.quantum import Dagger, Operator, Commutator
+
+   a = Operator('a')
+   adag = Dagger(a)
+
+.. code-block:: python
+
+   from sympy.physics.quantum import Ket
+   n = Ket('n')
+   print(a * n)
+   print(adag * n)
+
+.. code-block:: python
+
+   from sympy.physics.quantum.boson import BosonOp
+   from sympy.physics.quantum import Dagger, Commutator
+
+   a = BosonOp('a')
+   adag = Dagger(a)
+   print(Commutator(a, adag).doit())
+
+Visualization
+=============
+
+.. image:: ../../../_static/creation_annihilation_ladder.png
```

Comment:
We don't include png's into the source tree. Please make these svg's if hand drawn or use the matplotlib extension to produce figures from the code. See the other tutorials for examples of this.